plugins/source/aws/client/resolvers.go

```diff
@@ -139,15 +139,14 @@ func MarshaledJsonResolver(path string) schema.ColumnResolver {
 		var j map[string]interface{}
 		field := funk.Get(r.Item, path, funk.WithAllowZero())
 
+		if field == nil {
+			return nil
+		}
```

Comment:
I would suggest we set the value to `nil`... that way we can always be deterministic about `nil` vs `empty` vs `{}`